osgar/drivers/__init__.py

```diff
@@ -6,10 +6,12 @@
 from .simulator import SpiderSimulator
 from .logsocket import LogTCP, LogUDP, LogHTTP
 from .sicklidar import SICKLidar
+from .johndeere import JohnDeere
 
 # dictionary of all available drivers
 all_drivers = dict(gps=GPS, imu=IMU, spider=Spider, serial=LogSerial,
                    can=CANSerial, simulator=SpiderSimulator,
                    tcp=LogTCP, udp=LogUDP, http=LogHTTP,
-                   lidar=SICKLidar)
+                   lidar=SICKLidar,
+                   johndeere=JohnDeere)
```

Comment:
We might want to put the closing bracket on the next line so that it is easier to merge (less conflict). Or we can make `all_drivers` dynamic.